### PR TITLE
7 login user

### LINKED ISSUE #7: Login
```
As a user
So that I can access other features
I want to be able to login
```

### PR DESCRIPTION
**Pull request by Tagrand**
close #7

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -12,3 +12,4 @@ gem 'rack'
 gem 'rspec'
 gem 'rubocop'
 gem 'sinatra'
+gem 'sinatra-flash'
```
Comment (domvernon) [inline Gemfile]: This line of code is perfect. I really think you have adhered to SOLID principles and I can clearly see what it is doing.

File: app/models/user.rb [hunk 2]
```diff
@@ -4,4 +4,10 @@ class User
   property :id, Serial
   property :username, String
   property :password, BCryptHash
+
+  def self.authenticate(username, password)
```
Comment (domvernon) [inline app/models/user.rb]: BCrypt might do this out of the box, but we can explore this later.

**Comment by domvernon** [comment]
:shipit: in case the above wasn't clear!

**Comment by GiadaSimonetti** [comment]
:shipit:

**Pull request merged**